vmm/src/vmm_config/logger.rs

```diff
@@ -32,19 +32,31 @@ pub struct LoggerConfig {
     /// Named pipe used as output for metrics.
     pub metrics_fifo: String,
     /// The level of the Logger.
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub level: Option<LoggerLevel>,
+    #[serde(default = "default_level")]
+    pub level: LoggerLevel,
     /// When enabled, the logger will append to the output the severity of the log entry.
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub show_level: Option<bool>,
+    #[serde(default = "default_show_level")]
```

Comment:
For types that implement `Default` (such as the primitive type `bool`) u can leave the attribute as simply `#[serde(default)]` if you are happy with the provided value (which is `false` in this case).